src/Microsoft.AspNetCore.Mvc.Analyzers/ApiControllerFacts.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Mvc.Analyzers
+{
+    public static class ApiControllerFacts
```

Comment:
LEL?

Reply:
If we're not actually testing anything, let's not add this project. We have plenty of projects

Reply:
Oops